src/named.rs

```diff
@@ -208,9 +208,9 @@ pub enum NamedChain {
     #[strum(to_string = "mantle")]
     #[cfg_attr(feature = "serde", serde(alias = "mantle"))]
     Mantle = 5000,
-    #[strum(to_string = "mantle-testnet")]
-    #[cfg_attr(feature = "serde", serde(alias = "mantle-testnet"))]
-    MantleTestnet = 5001,
+    #[strum(to_string = "mantle-sepolia")]
+    #[cfg_attr(feature = "serde", serde(alias = "mantle-sepolia"))]
+    MantleSepolia = 5003,
```

Comment:
please keep the old variant name and add aliases to avoid breaking changes

Reply:
It is a bit confusing because the `Mantle Testnet` was `5001` and the new `Mantle Sepolia Testnet` is `5003` see: https://chainlist.org/?search=mantle&testnets=true

Decided to add back `MantleTestnet` for backwards compatibility with `Mantle Sepolia Testnet` being additive